helpers/installcredprovider.sh

```diff
@@ -3,12 +3,16 @@
 # of the artifacts credential provider plugin for dotnet and
 # install it into $HOME/.nuget/plugins.
 # SEE: https://github.com/Microsoft/artifacts-credprovider/blob/master/README.md
+#
+# To install a release other than `latest`, set the `AZURE_ARTIFACTS_CREDENTIAL_PROVIDER_VERSION` environment
+# variable to match the name of a supported release.
+# SEE: https://github.com/microsoft/artifacts-credprovider/releases
 
 REPO="Microsoft/artifacts-credprovider"
 FILE="Microsoft.NuGet.CredentialProvider.tar.gz"
-VERSION="latest"
+: ${AZURE_ARTIFACTS_CREDENTIAL_PROVIDER_VERSION="latest"}
 # URL pattern documented at https://help.github.com/en/articles/linking-to-releases as of 2019-03-29
-URI="https://github.com/$REPO/releases/$VERSION/download/$FILE"
+URI="https://github.com/$REPO/releases/${AZURE_ARTIFACTS_CREDENTIAL_PROVIDER_VERSION}/download/$FILE"
```

Comment:
It looks like the download URL is different for latest vs other versions. 

Download for latest is: 
`https://github.com/$REPO/releases/latest/download/$FILE` 
and download specific version you need to use the tag name and
`https://github.com/$REPO/releases/download/${version_tag_name}/$FILE`
🤷 

At least I wasn't able to get the first one working with a specific version. 

If you want to get your contribution in, I'm happy to approve and merge this PR after you fix the URL issue if you get it done before Monday. We're releasing a new version of the cred provider on Monday and I'm going to merge my [other PR](https://github.com/microsoft/artifacts-credprovider/pull/302) before doing that. And in that other PR I added the version support as well but I'm happy to merge your PR first if it's ready by the time of the release. :)